pkg/platform/kube/functionres/lazy.go

```diff
@@ -622,20 +643,7 @@ func (lc *lazyClient) createOrUpdateDeployment(functionLabels labels.Set,
 			return nil, err
 		}
 
-		body, err := json.Marshal(deployment)
-		if err != nil {
-			return "", errors.Wrap(err, "Failed to marshal deployment resource")
-		}
-
-		deployment, err = lc.kubeClientSet.AppsV1beta1().Deployments(function.Namespace).Patch(deployment.Name,
-			types.MergePatchType,
-			body)
-		if err != nil {
-			return nil, errors.Wrap(err, "Failed to patch the updated deployment")
-		}
-
-		// we must call it after, as marshal will omit the deployment's rollingUpdate field change to nil (omitempty)
-		return lc.updateDeploymentStrategy(deployment, function)
```

Comment:
You removed the handling of deployment strategy in case the function has GPUs

Reply:
Added it back (with slight changes)